extension/scripts/background.js

```diff
@@ -33,8 +33,8 @@ chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
 				)
 			})
 		})
-	} else if (message.row) {
-		const body = JSON.stringify({row: message.row})
+	} else if (message.content) {
+		const body = JSON.stringify({content: message.content})
 		console.log('4. Message body sent: ', body)
 		fetch('http://localhost:3001/send-image', {
```

Comment:
Eventually we'd want to make the base URL + port into an environment variable, but this is fine for now.